enterprise/server/soci/soci.go

```diff
@@ -0,0 +1,316 @@
+package soci
```

Comment:
nit: can we call this `socistore` or something like that? this package is implementing a specific API with a very specific surface and i don't think the current name conveys that.

Reply:
Changed to `sociartifactstore` -- I didn't want to use `socistore` because that's the name of the binary that implements the fs that podman reads from. Open to other suggestions though.